themes/bootstrap3/templates/myresearch/mylist.phtml

```diff
@@ -46,9 +46,14 @@
             </div>
           <? endif; ?>
         <? elseif ($recordTotal > 0): ?>
-          <?=$this->transEsc("Showing")?>
-          <strong><?=$this->localizedNumber($this->results->getStartRecord())?></strong> - <strong><?=$this->localizedNumber($this->results->getEndRecord())?></strong>
-          <?=$this->transEsc('of')?> <strong><?=$this->localizedNumber($recordTotal)?></strong>
+          <?
```

Comment:
For some reason, the "showing x-y of z" only shows up when you're viewing your combined favorites (/MyResearch/Favorites). When you are viewing an individual list, there is no status indication at all -- just the edit/delete buttons when logged in, and nothing when logged out and viewing a public list. Is this intentional, or the result of an oversight? (Note that this issue is not directly related to translation changes -- I just happened to notice it while testing).